rfcs/0001-rfcs/proposal.md

```diff
@@ -0,0 +1,122 @@
+---
+author: Roni Choudhury
+date: 2020-06-10
+---
+
+# RFC: Process for Creating RFCs
+
+This is the first official Multinet RFC. It lays out the procedure for creating
+RFCs, soliciting feedback, and making them official.
+
+## Motivation
+
+We have historically made use of Google Docs to put together proposals for
+technical work that is either contentious, non-obvious, or both. These written
+proposals enable the author to clarify their thinking on a topic, write down at
+least minimal detail about the topic to transmit to others on the team, and
+formulate an action plan once the proposal is accepted.
+
+However, Google docs have some drawbacks: they are not particularly transparent
+and trackable; they are difficult to comment on for deeper discussions; and they
+are less pleasant to read and write than simple markdown files.
+
+Therefore, this proposal lays out a procedure for creating markdown-based RFCs,
+getting feedback and generating discussions for them, and settling on accepted
+versions for use in creating execution plans.
+
+## Proposals
+
+### Definitions
+
+- **RFC.** A "request for comments", though the term is used to mean both a
+  proposal for how something should be done, and as a de facto standard for how
+  that thing is done. We are avoiding the term **proposal** since that word is
+  overloaded to mean "grant proposal" in an academic context.
+
+### Procedure for Creating a new RFC
+
+1. Clone the RFCs repository with a command such as `git clone https://github.com/multinet-app/multinet-rfcs`.
+2. Create a new branch naming your RFC: `git checkout -b rfc/my-awesome-proposal`.
+3. Push your branch: `git push -u origin rfc/my-awesome-proposal`.
```

Comment:
Nope!

A branch is just a pointer to a commit. When you create a local branch, it has no corresponding remote branch yet, so pushing simply creates that remote.

I make a habit of doing this so that any local commits I do make are properly shown as being behind the corresponding remote. That's a good reminder to push in order to synchronize myself as soon as my local branch is ahead of the remote (and therefore in potential danger of being lost if my computer explodes).